admission_control/admission-control-service/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub mod mocks;
 /// AC runtime to launch gRPC and network service
 pub mod runtime;
 /// Handler for sending transaction write requests upstream if needed
-mod upstream_proxy;
+pub mod upstream_proxy;
```

Comment:
nit: can we keep it private and only expose structures that we really need to be public